website/content/docs/waypoint-hcl/app.mdx

```diff
@@ -121,8 +121,8 @@ The `app` stanza is composed of other stanzas and parameters, namely `build`,
   behavior for this application. If this isn't specified, the default settings
   configured on the Waypoint server will be used.
 
-[build] /docs/waypoint-hcl/build 'Build Stanza'
-[deploy] /docs/waypoint-hcl/deploy 'Deploy Stanza'
-[release] /docs/waypoint-hcl/release 'Release Stanza'
-[runner] /docs/waypoint-hcl/runner 'Runner Stanza'
-[url] /docs/waypoint-hcl/url 'url Stanza'
+[build]: /docs/waypoint-hcl/build 'Build Stanza'
+[deploy]: /docs/waypoint-hcl/deploy 'Deploy Stanza'
+[release]: /docs/waypoint-hcl/release 'Release Stanza'
+[runner]: /docs/waypoint-hcl/runner 'Runner Stanza'
+[url]: /docs/waypoint-hcl/url 'url Stanza'
```

Comment:
Wow ran into this today, ty! Just a a bystander note (not a reviewer), i _think_ the links just need to have "/waypoint" prepended to i.e. 
```suggestion
[build]: /waypoint/docs/waypoint-hcl/build 'Build Stanza'
[deploy]: /waypoint/docs/waypoint-hcl/deploy 'Deploy Stanza'
[release]: /waypoint/docs/waypoint-hcl/release 'Release Stanza'
[runner]: /waypoint/docs/waypoint-hcl/runner 'Runner Stanza'
[url]: /waypoint/docs/waypoint-hcl/url 'url Stanza'
```
to make vercel happy 😄